frontend/app/controllers/Joiner.scala

```diff
@@ -138,6 +138,7 @@ trait Joiner extends Controller {
         eventId <- PreMembershipJoiningEventFromSessionExtractor.eventIdFrom(request)
         event <- EventbriteService.getBookableEvent(eventId)
       } yield {
+        EventbriteService.getService(event).wsMetrics.put("user-sent-to-eventbrite", 1)
```

Comment:
okay @jennysivapalan – have renamed the metric, and also added the tier (to both eventbrite metrics) on @wpf500's advice. This means the radiator will stop showing new data once this goes out, but we can do a corresponding update there too. And also break down signups by tier/intent!